CMakeTests/FindOProfile.cmake

```diff
@@ -0,0 +1,33 @@
+# - Try to find OProfile
+# Once done this will define
+# OPROFILE_FOUND - System has OProfile
+# OPROFILE_INCLUDE_DIRS - The OProfile include directories
+# OPROFILE_LIBRARIES - The libraries needed to use OProfile
+
+set(FIND_OPROFILE_PATHS
+    /usr/local
+    /usr
+    /sw
+    /opt/local
+    /opt)
```

Comment:
How does this behave on OSX, if at all? IIRC, port puts its stuff somewhere under /opt